relay-server/src/metrics_extraction/transactions/types.rs

```diff
@@ -62,6 +68,12 @@ impl IntoMetric for TransactionMetric {
                 MetricUnit::Duration(unit),
                 tags.into(),
             ),
+            TransactionMetric::DurationCoarse { unit, value, tags } => (
+                Cow::Borrowed("duration_coarse"),
```

Comment:
open to better naming suggestions https://github.com/getsentry/sentry/pull/56202#discussion_r1330333443

Reply:
We sadly do not have a naming scheme for such metrics yet, and likely over time they will rather become a materialization in storage and would be fully hidden in the namespace.

`duration_coarse` sounds good to me. It would allow us to add other uncontroversial tags without introducing a new metric every time. 

@jjbayer any thoughts on this?

Reply:
No strong preference, but we do already have a similar metric for spans that we suffixed with `_light` instead of `_coarse`, so I would use the same suffix for consistency: https://github.com/getsentry/relay/blob/4d0650ee8a2cfa70adaef2a51774f266857ab367/relay-dynamic-config/src/defaults.rs#L100